pkg/storage/cacher_test.go

```diff
@@ -246,10 +247,12 @@ func TestWatch(t *testing.T) {
 	}
 	defer nowWatcher.Stop()
 
+	t.Log("verifying")
 	verifyWatchEvent(t, nowWatcher, watch.Added, podFooPrime)
 
 	_ = updatePod(t, etcdStorage, podFooBis, fooUpdated)
 
+	t.Log("verifying")
```

Comment:
I'd like to keep these. If the verify calls fail, there is no way to tell which one failed